### ISSUE TITLE
[BUG]: Wrong nickname or password. Try again or try resetting your password

### ISSUE THREAD

**Issue by riczescaran** [azukaar/Cosmos-Server#30]
### What happened?

This problem keeps happening even though I am putting the correct password.

### What should have happened?

When encountering this problem It should ask for password reset function. 

### How to reproduce the bug?

Configured my /etc/hosts

`127.0.0.1 homelab.local`

My docker compose file:

```
version: "3.3"
services:
    cosmos-server:
        ports:
            - "80:80"
            - "443:443"
        container_name: cosmos-server
        hostname: cosmos-server
        restart: always
        volumes:
            - "/var/run/docker.sock:/var/run/docker.sock"
            - "./config:/config"
            - "./host:/mnt/host"
        image: "azukaar/cosmos-server:latest"
```

Steps I did when setting up the cosmos server:
1. Automatically create a secure database (recommended)
2. Use HTTP only (not recommended)
3. Set hostname homelab.local. I checked "Use Wildcard Certificate for *.homelab.local" and clicked update
4. Supplied username and password (I used lastpass password generator)

### Other details

HTTP response:
```
{
    "status": "error",
    "message": "User Logging Error",
    "code": "UL001"
}
```

### System details

- OS: Windows 10 Pro
- Browser Microsoft Edge Version 114.0.1823.43 (Official build) (64-bit)]
- Docker version 4.20.1

**Comment by azukaar** [comment]
This behaviour is usually a symptom that the Database is not properly setup. This could have multiple reasons, maybe you ran the installer twice without flushing all the configs, maybe something in you setup messed with the containers (ex. Unraid) maybe the password got set differently by something.
Either way, could yo please dive into the logs of both Cosmos (to try to find some connection error to the DB) and MongoDB, to make sure the DB is healthy

**Comment by riczescaran** [comment]
Thanks for the response. I will give you feedback once I’m done checking the logs.

**Comment by riczescaran** [comment]
@azukaar Upon checking it again It seems that there is no connection errors to the DB. 

I noticed that it also creates multiple DB instances that is also related with @t984447 problem. 

I will try to pull the latest image to see if still persists. Thank you!

**Comment by azukaar** [comment]
Sounds good! Make sure the network tab of the browser console is visible so we can see those potential duplicated requests if they exist :)

**Comment by riczescaran** [comment]
@azukaar Hello, sorry for the late feedback about the screen record. Here's the video. Please let me know if you found something.


https://github.com/azukaar/Cosmos-Server/assets/12409127/06385a72-e4c7-4083-a9fa-552f05fdb986

**Comment by riczescaran** [comment]
@azukaar here's my second attempt. It seems that the user was created to the wrong DB container.
I think once the DB connection string for MongoDB initialized to **cosmos.config.json** it should not be updated.

https://github.com/azukaar/Cosmos-Server/assets/12409127/57530a76-1dbc-4837-9eda-e36e51fdd3fa

**Comment by azukaar** [comment]
Hello @riczescaran  @t984447 ! First of all thanks you all for your patience and your help on this.
Thanks to the infos you provided I was able to provide a fix for the issue in 0.7.9 !
Please try it when convenient and let me know how it goes.

If you are curious about the details: turns out some browser don't like it when a server take too long to reply to a request (ex. when Cosmos wait on the DB to start) So they re-send the HTTP request multiple time. I have converted the request so that Cosmos stream some logs while the browser is waiting so that the retry does not happen anymore.

Thanks again!
